po/zh_CN.po

```diff
@@ -561,9 +598,13 @@ msgstr "放大(_I)"
 msgid "Zoom _Out"
 msgstr "缩小(_O)"
 
+#: data/menu.ui.h:41
+msgid "缩放适合(_F)"
+msgstr ""
```

Comment:
I think you accidentally overwrote the msgid here.

Thats probably `msgid "Zoom _Fit"`

Reply:
I'm so sorry, I've re-added the missing characters. Thanks for your thoughtful reminder.